### ISSUE TITLE
Doesn't open at all in OSX Mojave 10.14, no errors

### ISSUE THREAD

**Issue by csgyuricza** [wiztools/rest-client#78]
Even after allowing it to open (in the Security & Privacy screen), the app won't open. The icon bounces in the dock but that's it. No messages or errors. Tried version 3.7.0.

**Comment by csgyuricza** [comment]
Java is 1.8 (1.8.0_181-b13)

**Comment by csgyuricza** [comment]
I realize now this is a duplicate of https://github.com/wiztools/rest-client/issues/77.

**Comment by subwiz** [comment]
(cross posting from #77 )

I've modified the code and dependencies to support Java 8. A build of this version is available here:

* http://download.wiztools.org/rest-client/restclient-ui-fat-3.7.1-SNAPSHOT.jar
* http://download.wiztools.org/rest-client/restclient-ui-3.7.1-SNAPSHOT.dmg

Can the audience test the builds and let me know if it works as expected? Especially in Mac I want help knowing if the `.dmg` version works with both Java 8 (with no Java 10 installed in parallel) and Java 10.

Once this sanity confirmation is given by the community, I'll publish the release.
